src/audioPlayer.js

```diff
@@ -0,0 +1,96 @@
+// Instantiate the PLAYER variable in the Webflow page's head since it is needed across multiple scripts.
+PLAYER = document.querySelector('[data-element=audio-player]')
+PLAYER.querySelector('source').src = '0'
+
+const AUDIO_TOGGLES = document.querySelectorAll('[data-element=player-toggle]')
+
+AUDIO_TOGGLES.forEach((podcastTrigger) => {
+    podcastTrigger.addEventListener('click', setPlayer.bind(podcastTrigger))
+})
+
+// Helper to safely call a function declared in the analytics script that should be loaded.
+function getAnalyticsEventProperties(eventName, dataset, podcastTrigger) {
```

Comment:
Wondering if we send a Sentry alert here, as it should never happened if the scripts are properly configured